### PR TITLE
GA 7.12 Release Notes

### PR DESCRIPTION
**Pull request by florentsarat**

### CODE REVIEW

File: md/release-notes.md [hunk 1]
```diff
@@ -4,6 +4,13 @@
 **Note:** The 7.12 is currently work-in-progress (WIP). The 7.12.0 GA is planned for December 2020.
 :::
 
+## Reinforcing our Open Source DNA
+Bonita has always a highly open-source project and we strongly that it is the best to create a great and extensible platform. 
+
+In 7.12, we've decide to provide the community edition all the tools to create state of the art and successful automation projects.
```
Comment (NathalieC) [inline md/release-notes.md]:
```suggestion
In 7.12, we've decide to provide the Community edition with all the tools to create state of the art and successful automation projects.
```

Comment (rbioteau) [reply md/release-notes.md]: decide**d**

Comment (NathalieC) [reply md/release-notes.md]: good catch!

File: md/release-notes.md [hunk 2]
```diff
@@ -4,6 +4,13 @@
...
+
+More information here
```
Comment (rbioteau) [inline md/release-notes.md]: Was a link expected here or are you referring to the section below?

File: md/release-notes.md [hunk 3]
```diff
@@ -83,6 +90,9 @@ In the Community edition, XPDL and jBPM importers have been removed.
 In Enterprise edition, Visio and Aris importers have been removed.  
 Use BPMN 2 importer for model exchange with third-party editors.
 
+## Translations
+Please note that Bonita 7.12.0 is not fully translated in Japanese. 
+Translations will be added in the next maintenance versions.
```
Comment (rbioteau) [inline md/release-notes.md]: will -> might

File: md/release-notes.md [hunk 4]
```diff
@@ -82,8 +87,16 @@ Search keys can be defined and used in the community edition. Take a look at the
 ## Feature deprecations and removals
 
 ### Deprecations
+#### Bonita Portal transformation
+Bonita Portal is being transformed into Bonita Applications. When Bonita Applications are ready, Bonita Portal will be removed. Developers and users will need to learn how to stop using the Portal and start using Bonita Applications instead. This change will allow Bonita and its users to get free from Google Web Toolkit (GWT) technology and offer opportunities for customization.
+Indeed, some Portal pages (built with GWT) are being totally recreated with our own UI Designer. They will be customizable. Other pages (those that were already using another technology than GWT) are being wrapped and will not be customizable.
+Moreover, as any Living Application, Bonita applications will be extensible to add any page the users need. More details in the upcoming versions of Bonita.
+Until then, we strongly advise not to create Custom Portal Profiles anymore but applications instead if possible. When Bonita Portal does not exist anymore, the existing Portal Custom Profiles will need to be migrated into Living applications.
```
Comment (abirembaut) [inline md/release-notes.md]: - I would remove the "if possible"
- maybe "When Bonita Portal **will not** exist" or "When Bonita Portal **will be replaced with applications**"

File: md/release-notes.md [hunk 5]
```diff
@@ -82,8 +87,16 @@ Search keys can be defined and used in the community edition. Take a look at the
...
+
 
 ### Removals
+#### IE11 support
+Internet Explorer 11 is not supported anymore through the Bonita Platform.
```
Comment (abirembaut) [inline md/release-notes.md:123]: Maybe we can add. "The web browsers supported by Bonita 7.12 are Mozilla Firefox, Google Chrome and Microsoft Edge."
Or something like that...

**Review by abirembaut — COMMENTED** [review]
Jut 2 comments regarding the portal deprecation and ie11

File: md/release-notes.md [hunk 6]
```diff
@@ -41,19 +53,27 @@ With the autocomplete widget you have a returned value that different from the d
 
 ### Runtime changes
 
-#### Micrometer updated to 1.6.0
```
Comment (NathalieC) [inline md/release-notes.md:44]: Shouldn't that be part of the Technical updates below?

File: md/release-notes.md [hunk 7]
```diff
@@ -41,19 +53,27 @@ With the autocomplete widget you have a returned value that different from the d
...
+#### Micrometer updated to 1.6.1
 This version of micrometer adds new metrics. Notably two new hibernate metrics: `hibernate.cache.query.plan` respectively `hit` & `miss`, to help troubleshoot performance problems.
 How to activate this metrics is documented [here](runtime-monitoring.md).
 
-
 #### Fault tolerance mechanisms
```
Comment (NathalieC) [inline md/release-notes.md]: Is "fault tolerance" or "recovery" the right wording to use everywhere?

File: md/release-notes.md [hunk 8]
```diff
@@ -62,7 +82,6 @@ Same thing if you have a logout link in a custom page that does not pass a `logi
 If you use Bonita layout version 5 or a customized version of it in your applications, make sure you upgrade to version 6 when migrating. Otherwise, the logout button will not redirect to the login page when clicked.
 
 #### Search keys
```
Comment (NathalieC) [inline md/release-notes.md:99]: Isn't that already part of the Reinforce Open Source DNA chapter at the top of the page?

File: md/release-notes.md (hunk 6 above)
Comment (pabloalonso) [reply md/release-notes.md:44]: It explain the new metric added and that we contributed to this release

**Pull request merged**